php-checks/src/main/java/org/sonar/php/checks/regex/StringReplaceCheck.java

```diff
@@ -46,6 +59,12 @@ public void checkRegex(RegexParseResult regexParseResult, FunctionCallTree regex
     }
   }
 
+  private static boolean limitParameterIsNotUsedOrSetToDefault(FunctionCallTree regexFunctionCall) {
```

Comment:
Maybe we can keep the name of this parameter more generic here, as in the `checkFunctionCall` caller method.
```suggestion
  private static boolean limitParameterIsNotUsedOrSetToDefault(FunctionCallTree tree) {
```